routes/users.js

```diff
@@ -54,13 +54,14 @@ router.patch(
 // upload.single('profile') -> multer inmemory storage of file for type multipart/form-data
 router.post("/picture", authenticate, checkIsVerifiedDiscord, upload.single("profile"), users.postUserPicture);
 router.patch(
-  "/picture/verify/:id",
+  "/picture/verify/:userId",
   authenticate,
   authorizeRoles([SUPERUSER]),
   userValidator.validateImageVerificationQuery,
   users.verifyUserImage
 );
-router.get("/picture/:id", authenticate, authorizeRoles([SUPERUSER]), users.getUserImageForVerification);
+router.get("/picture/all", authenticate, authorizeRoles([SUPERUSER]), users.getAllUsersPhotoVerificationRequests);
+router.get("/picture/:userId", authenticate, users.getUserPhotoVerificationRequests);
```

Comment:
Why superuser requirement is removed?

Reply:
because this API can be used for non-SUPERUSER to get their own data and only theirs.